alacritty.yml

```diff
@@ -481,10 +482,13 @@
 #
 #   - ToggleViMode
 #   - SearchForward
+#       Start searching toward the right of the search origin.
 #   - SearchBackward
+#       Start searching toward the left of the search origin.
 #   - Copy
 #   - Paste
 #   - PasteSelection
+#       Paste from the selection clipboard buffer on Wayland/X11.
```

Comment:
Can we move this down to `CopySelection`? This already states that it's Linux/BSD only, so we can then change it to `Paste from the selection clipboard buffer.`

Reply:
Fixed in next commit. I'm using the term "selection buffer" for both, rather than "selection clipboard buffer". If a Linux user is reading alacritty.yml file to set up custom key bindings, they likely understand what the selection buffer is.

Reply:
Sounds good to me.